### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Disk_DiskInfoTable_Screen reader: Voice over is not announcing table rows information when user navigates with Ctrl + Option + Arrow keys.

### ISSUE THREAD

**Issue by keerthiduvvuri** [microsoft/AzureStorageExplorer#7617]
### Preflight Checklist

- [X] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [X] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [X] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.32.1(96)

### Regression From

NO

### Architecture

x64

### Storage Explorer Build Number

20231114.10

### Platform

macOS

### OS Version

MAC Version 13.6.1

### GitHub Tags:
#A11yMAS;#A11yTCS;#MAC;#DesktopApp;#A11ySev2;#BM_AzureToolsStorageExplorer_MAC_Dec2023;#Azure Tools Storage Explorer;#FTP;#Voiceover;#WCAG1.3.1;#Info and Relationship;#Element:Table;

### Environment Details:
Application: Microsoft Azure Storage Explore
version: 1.32.1 (96)
Screen Reader: Voice Over

### Steps to Reproduce

1. Launch Storage Explorer
2. Tab Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Tab Navigate to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Tab Navigate to "Disks" of the selected subscription.
5. Tab Navigate to "A11ytest" and click on it.
6. Tab Navigate to Table under the tool bar.
7. Turn on Voice over and Observe that Voice over is not announcing table rows information when user navigates with Ctrl + Option + Arrow keys.

### Actual Experience

Voice over is not announcing table rows information when user navigates with Ctrl + Option + Arrow keys.

### Expected Experience

Voice over should be announcing table rows information when user navigates with Ctrl + Option + Arrow keys.

### User Impact:
Screen reader users will face difficulty if Voice over is not announcing table rows information when user navigates with Ctrl + Option + Arrow keys.

### Attachment:
![A11y_Azure Tools Storage Explorer_Disk_DiskInfoTable_Screen reader](https://github.com/microsoft/AzureStorageExplorer/assets/98728920/6a9040bb-424f-4e76-abb1-d40c38391c19)

**Comment by richardMSFT** [comment]
This is fixed in the new react managed disks explorer that has been merged.


https://github.com/microsoft/AzureStorageExplorer/assets/111301956/c533fccc-e15e-482b-baf4-6e654c59d2e2

**Comment by keerthiduvvuri** [comment]
@richardMSFT Can you please let us know in which version this bug is fixed, so that i can verify and close it.

**Comment by richardMSFT** [comment]
Hi @keerthiduvvuri. The fix has been merged, but a build with it hasn't been made yet.

**Comment by kupatkar99** [comment]
Verified the issue in below environment:
Azure Tools Storage Explorer(Version: 1.33.0-dev (96))
Issue is fixed. Hence Closing the bug. Please find added attachment for reference.
<img width="1440" alt="7617_Fixed" src="https://github.com/microsoft/AzureStorageExplorer/assets/98530069/a38701a3-3902-4864-a6f6-3c8810b7322b">

**Comment by keerthiduvvuri** [comment]
#Closed;
